### DISCUSSION TITLE
Online Only Meetings

### DISCUSSION THREAD

**Discussion by ljread** [code4recovery/12-step-meeting-list#1064] [Q&A]
Our area has several meetings that were physical meetings before COVID but are still online only. These are marked temporarily closed but still retain their location information for when they re open. 

Recently I noticed that this prevents these meeting from being listed on the PDF generated using the 12 Step Meeting PDF Generator by Chris T. (I realize/understand that this is separate from C4R.) But these meeting will be listed if their location information is removed. To save it for future use, I copied this info to the locations notes field. 

The problem this created was that all of the online only meeting in a specific region (we use this as cities/towns) are then linked together and the location notes info is overwritten - even when *not*checking the box to update all meeting at that location. The only work around I have found is to link the meetings at a given location as a group and add the location info for future use in the group notes field.

Anyone else experienced this? Am I doing something wrong? Is there a better work around?

Thankfully, this only affected a few meetings before I realized the behavior. 

We are on the current version of the TSML (3.14.12)

Thanks for any help/insights,

John

**Comment by joshreisner** [comment]
hi john,

location notes are shared between all meetings at a given "address" which could either be an approx location like `Boston, MA, USA` or a street address.

so if you want to retain unique notes they should go in the regular meeting notes field, even though i realize they pertain to the "location" -- because regular notes can differ between meetings that share a location

we generally recommend considering meetings that still aren't meeting in person after three years as "online" meetings now, and they can always be edited if they return to in-person or go hybrid
